packages/storybook/stories/drawers/Drawer.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import type { Meta, StoryObj } from '@storybook/react';
```

Comment:
Are we expecting more types of Drawers?
If not I'd place it in the root to avoid the double folder structure.